packages/color-helpers/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changes to Color Helpers
 
-### Unreleased
+### Unreleased (major)
```

Comment:
I don't know if this will be picked up by the release plan since it needs the --public flag the first time

Reply:
Good catch! I've modified the publishing script to handle this.